### ISSUE TITLE
Update method doesn't trigger in IE11

### ISSUE THREAD

**Issue by mikestreety** [kevinongko/vue-numeric#43]
Hey,

I'll come back and properly fill out this issue this afternoon, but thought I would just put it here before I forget.

The `this.update` method doesn't seem to ever get fired in **IE11** or below. However the `this.process` one does.

I have temporarily updated the `this.process` method to the following (i expanded the if statements while I was debugging) and remove the `update` method

```js
/**
* Validate value before update the component.
* @param {Number} value
*/
process: function(value) {
	if (value >= this.max) {
		value = this.max;
	}
	if (value <= this.min) {
		value = this.min;
	}
	if (!this.minus && value < 0) {
		value = this.min >= 0 ? this.min: 0;
	}

	this.$emit('input', Number(accounting.toFixed(value, this.precision)));
},
```

**Comment by kevinongko** [comment]
Hi @mikestreety,

I tried the live demo with IE11, and it's works correctly, could you give steps to reproduce?
Thanks

**Comment by mikestreety** [comment]
This pen, in IE11, doesn't work for me:

https://codepen.io/mikestreety/pen/aLgMPM

Video: http://recordit.co/0sJs5eLcJj

**Comment by mikestreety** [comment]
But the live demo does work 🤔

**Comment by tjrchampion** [comment]
Any follow up on this? Experiencing the exact same problem in IE 11.

**Comment by kevinongko** [comment]
Fixed in 2.2.7
